includes/class-wc-moip-gateway.php

```diff
@@ -652,20 +660,41 @@ protected function get_payment_xml( $order ) {
 
 			// Installments info.
 			if ( 'yes' == $this->installments ) {
-				$installments = $instruction->addChild( 'Parcelamentos' );
-				$installment = $installments->addChild( 'Parcelamento' );
-				$installment->addChild( 'MinimoParcelas', $this->installments_mininum );
-				$installment->addChild( 'MaximoParcelas', $this->installments_maxium );
-				$installment->addChild( 'Recebimento', $this->installments_receipt );
-
-				if ( ! empty( $this->installments_interest ) && $this->installments_interest > 0 ) {
-					$installment->addChild( 'Juros', str_replace( ',', '.', $this->installments_interest ) );
+				$installments_mininum = $this->installments_mininum;
+				$installments_maxium = $this->installments_maxium;
+
+				//Check if the number of installments respects the minimun amount rule
+				if ($this->installments_minimum_amount) {
```

Comment:
Not following WordPress coding standards: https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/